spark/v3.5/build.gradle

```diff
@@ -258,6 +258,7 @@ project(":iceberg-spark:iceberg-spark-runtime-${sparkMajorVersion}_${scalaVersio
     integrationImplementation "org.apache.spark:spark-hive_${scalaVersion}:${libs.versions.spark35.get()}"
     integrationImplementation libs.junit.vintage.engine
     integrationImplementation libs.junit.jupiter
+    integrationImplementation libs.junit.platform.launcher
```

Comment:
I'm curious why this isn't needed in the Spark 3.4 runtime project?

Reply:
nvm. This is because currently the Spark 3.4 integration test is still a JUnit4 test